### ISSUE TITLE
Please tag new release.

### ISSUE THREAD

**Issue by onlyjob** [projectatomic/oci-systemd-hook#104]
It's been a while since v0.1.5 released on 2017-02-08. Let's tag a new release please?

**Comment by rhatdan** [comment]
@onlyjob Good point.  I would like to wait for https://github.com/projectatomic/oci-systemd-hook/pull/103 but if it seems like  it will be a while I will just cut a release and then another new one once it gets merged.
